src/main/java/de/uni_tuebingen/ub/nppm/db/SaveHelper.java

```diff
@@ -43,8 +44,42 @@ public static void updateAttribute(String table, String attribute, String value,
         updateAttribute(table, attribute, value, Integer.parseInt(id));
     }
 
+    public static void updateAttribute(String table, String attribute, String value, String formularAttribut, int id, String cond) throws Exception {
+        try ( Session session = getSession()) {
+            session.getTransaction().begin();
+            NativeQuery query = session.createNativeQuery("UPDATE " + table + " SET " + attribute + "= :value WHERE " + formularAttribut + "= :id" + cond);
+            query.setParameter("value", value);
+            query.setParameter("id", id);
+            query.executeUpdate();
+            session.getTransaction().commit();
+        }
+    }
+
+    public static void updateAttribute(String table, String attribute, String value, String formularAttribut, String id, String cond) throws Exception {
+        updateAttribute(table, attribute, value, formularAttribut, Integer.parseInt(id), cond);
+    }
+
+    public static void insertNewAttribute(String table, String attribute, String formularAttribut, String field, String value, int id, String Idvalue) throws Exception {
+        try ( Session session = getSession()) {
+            session.getTransaction().begin();
+
+            NativeQuery query = session.createNativeQuery("INSERT INTO " + table
+                    + " (" + attribute + ", " + formularAttribut + field + ")"
```

Comment:
All parts of the query must be escaped properly, not only id + value, but also attribute, formularAttribut and so on